File: app/src/main/java/com/kieran/winnipegbusbackend/SerializableLatLng.java
package com.kieran.winnipegbusbackend;

import com.google.android.gms.maps.model.LatLng;

import java.io.Serializable;

public class SerializableLatLng implements Serializable {
    private double latitude, longitude;

    public SerializableLatLng(LatLng latLng) {
        latitude = latLng.latitude;
        longitude = latLng.longitude;
    }

    public LatLng getLatLng() {
        return new LatLng(latitude, longitude);
    }
}

File: app/src/main/java/com/kieran/winnipegbusbackend/SerializableLatLng.kt
package com.kieran.winnipegbusbackend

import com.google.android.gms.maps.model.LatLng

import java.io.Serializable

class SerializableLatLng(latLng: LatLng?) : Serializable {
    val latitude: Double = latLng?.latitude ?: 0.0
    val longitude: Double = latLng?.longitude ?: 0.0

    val latLng: LatLng
        get() = LatLng(latitude, longitude)

}

